wordpress-application/task-definition.tf

```diff
@@ -0,0 +1,58 @@
+data "aws_iam_role" "ecs_role" {
```

Comment:
you need to create your own role.. this role only exists after running the ecs wizard on the console (on my case it failed as I have never used it before)

Reply:
`aws_iam_policy_attachment` added